new-player/src/androidTest/java/net/newpipe/newplayer/ExampleInstrumentedTest.kt

```diff
@@ -19,6 +19,6 @@ class ExampleInstrumentedTest {
     fun useAppContext() {
         // Context of the app under test.
         val appContext = InstrumentationRegistry.getInstrumentation().targetContext
-        assertEquals("net.newpipe.newplayer.test", appContext.packageName)
+        assertEquals("net.newpipe.newplayer", appContext.packageName)
```

Comment:
This file should just be removed, it's just an example test (and now it's failing on the CI)